cmd/cat/main.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"bufio"
+	"flag"
+	"fmt"
+	"log"
+	"os"
+)
+
+func main() {
+	var nFlag bool
+	flag.BoolVar(&nFlag, "n", false, "to number output lines")
+
+	flag.Usage = func() {
+		fmt.Println("Usage: cat [options] [file]")
+		fmt.Println("Options:")
+		flag.PrintDefaults()
+	}
+
+	flag.Parse()
+
+	filePath := flag.Arg(0)
```

Comment:
it should handle the case when no arguments are passed (check what cat command does)